packages/@vue/cli-plugin-vuex/prompts.js

```diff
@@ -0,0 +1,5 @@
+// these prompts are used if the plugin is late-installed into an existing
```

Comment:
Not necessary if empty.

Reply:
I wanted to add some prompts for snippets feature after I finish the hooks PR.

Reply:
Also, I want to make a PR where there's a single source of truth for the prompts. Right now, there are many duplicates.

Reply:
What do you want to refactor precisely? I'm doing a refactor on `@vue/cli` already.

Reply:
Some prompts in current repo are duplicated, once for `vue create` and once for `vue add @vue/{plugin}`. I want to move all those into cli-shared-utils and take them from there wherever necessary. Single Source of truth.

Reply:
It will be in the upcoming `@vue/cli-utils` package then.

Reply:
Will you take over that task as part of your refactor then?

Reply:
Yeah I can do it